src/main/java/org/apache/commons/imaging/formats/jpeg/decoder/Dct.java

```diff
@@ -282,15 +282,14 @@ public static void inverseDCT8(final float[] vector) {
         final float n1 = vector[0] - vector[4];
         final float n2 = b2 - a3;
         final float n3 = vector[0] + vector[4];
-        final float neg_n5 = neg_b4;
 
         // A2
         final float m3 = n1 + n2;
         final float m4 = n3 + a3;
         final float m5 = n1 - n2;
         final float m6 = n3 - a3;
-        // float m7 = n5 - n0;
-        final float neg_m7 = neg_n5 + n0;
+        // float m7 = b4 - n0;
+        final float neg_m7 = neg_b4 + n0;
```

Comment:
Quite sure these variables were copied from a paper or specification document. I would prefer to leave as in the document (if that's the case), but will have to take a better look later to verify if that's really the case.

Reply:
HI @kinow 
Changed
TY